R/set_args.R

```diff
@@ -54,6 +54,23 @@ set_legend_args <- function(object, fun.name, ..., legend.name) {
   
   paramsAll <- set_args(fun.name, list(...))
   
+  if(length(legend.name) > 1){
+    paramsAll_df <- as.data.frame(paramsAll, stringsAsFactors = FALSE)
+
+    for(p in seq(nrow(paramsAll_df))) {
```

Comment:
I think I was suggesting moving this loop to the get_legend_args and having it handle 1 or many legend.names, but I'm not sure the best approach to legend stuff at this point.